src/components/organisms/header/header.html

```diff
@@ -0,0 +1,112 @@
+<nav class="navbar navbar-expand-xl">
+  <a class="navbar-brand" href="#"> <img src="../../../img/header.png" /></a>
+
+  <button
+    class="navbar-toggler"
+    type="button"
+    data-toggle="collapse"
+    data-target="#nav-list"
+    aria-controls="nav-list"
+    aria-expanded="false"
+    aria-label="Toggle navigation"
+  >
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <!--ハンバーガーメニュー-->
+
+  <div class="collapse navbar-collapse" id="nav-list">
+    <ul class="navbar-nav me-auto">
+      <li class="nav-item active">
+        <a class="nav-link" href="#">HOME</a>
+      </li>
+      <div class="collapse navbar-collapse" id="nav-Dropdown">
+        <ul class="navbar-nav">
+          <li class="nav-item dropdown">
+            <a
+              class="nav-link dropdown-toggle"
+              href="#"
+              id="nav-Dropdown"
+              role="button"
+              data-toggle="dropdown"
+              aria-expanded="false"
+            >
+              CONCEPT
+              <div class="collapse-button">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="16"
+                  height="16"
+                  fill="currentColor"
+                  class="bi bi-plus-circle"
+                  viewBox="0 0 16 16"
+                >
+                  <path
+                    d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z"
+                  />
+                  <path
+                    d="M8 4a.5.5 0 0 1 .5.5v3h3a.5.5 0 0 1 0 1h-3v3a.5.5 0 0 1-1 0v-3h-3a.5.5 0 0 1 0-1h3v-3A.5.5 0 0 1 8 4z"
+                  />
+                </svg>
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="16"
+                  height="16"
+                  fill="currentColor"
+                  class="bi bi-dash-circle"
+                  viewBox="0 0 16 16"
+                >
+                  <path
+                    d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z"
+                  />
+                  <path
+                    d="M4 8a.5.5 0 0 1 .5-.5h7a.5.5 0 0 1 0 1h-7A.5.5 0 0 1 4 8z"
+                  />
+                </svg>
+              </div>
+            </a>
+
+            <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+              <li><a class="dropdown-item" href="#">ブログアーカイブ</a></li>
+              <li>
+                <a class="dropdown-item" href="#">お知らせアーカイブ</a>
+              </li>
+              <li>
+                <a class="dropdown-item" href="#">イベントアーカイブ</a>
+              </li>
+              <li>
+                <a class="dropdown-item" href="#">インタビューアーカイブ</a>
+              </li>
+              <li><a class="dropdown-item" href="#">FAQページ</a></li>
+              <li>
+                <a class="dropdown-item" href="#">固定ページ(デフォルト)</a>
+              </li>
```

Comment:
固定ページ(デフォルト)の後にCONCEPTが抜けています。
```suggestion
              <li>
                <a class="dropdown-item" href="#">固定ページ(デフォルト)</a>
              </li>
               <li><a class="dropdown-item" href="#">CONCEPT</a></li>
```